### DISCUSSION TITLE
Access another field from custom validation

### DISCUSSION THREAD

**Discussion by bomshteyn** [filamentphp/filament#6754] [Help] [answered]
I am using `laravel-validation-rules/credit-card` package to validate credit card entries.

To validate the CVC i also need to pass the `card_number`.

I have tried the following method:
```PHP
TextInput::make('card_number')
        ->columnSpanFull()
        ->label('Card Number')
        ->required()
        ->rules(['required', new CardNumber])
        ->mask(fn(Mask $mask) => $mask->pattern('0000 0000 0000 0000')),
 TextInput::make('CVC')
        ->label('CVC')
        ->required()
        ->rules([
            'required',
            'digits:3',
            new CardCvc(fn(\Closure $get) => $get('card_number'))
])
```

But am getting an error:
`Argument #2 ($subject) must be of type string, Closure given`

How can i pass it correctly to validate.

**Comment by pboivin** [comment, marked as answer]
Hi @bomshteyn , I think the `->rule()` method will let you access the `$get` helper:
```php
->rule(fn ($get) => new CardCvc($get('card_number')))
```

**Reply by bomshteyn** [reply]
@pboivin thank you! this works!
